meta-lmp-base/classes/image_types_ostree.bbclass

```diff
@@ -260,7 +261,12 @@ IMAGE_CMD:garagesign () {
                 push_success=1
                 break
             else
-                bbwarn "Push to garage repository has failed with errcode ${errcode}, retrying"
+                bbwarn "Push to garage repository has failed with errcode ${errcode}, retrying ${push_retries}/${GARAGE_PUSH_RETRIES}"
+                if [ "${GARAGE_PUSH_RETRIES_SLEEP}" -ne "0" ]; then
+                    sleep="$(($RANDOM%${GARAGE_PUSH_RETRIES_SLEEP}+1))""
+                    bbdebug 1 "Push to garage repository in ${sleep} seconds"
+                    sleep ${sleep}
+                fi
```

Comment:
i don't have a strong opinion on this change. not sure if it will actually help in practice or not.

otherwise, LGTM

Reply:
Yeah, seems ok, please propose upstream in meta-updater and we can move the review discussion there.

Reply:
I will do that after some tests on main-next